src/components/ResourceList/components/Item/Item.tsx

```diff
@@ -158,15 +159,16 @@ export class Item extends React.Component<CombinedProps, State> {
           testID="LargerSelectionArea"
         >
           <div onClick={stopPropagation} className={styles.CheckboxWrapper}>
-            <Checkbox
-              testID="Checkbox"
-              id={this.checkboxId}
-              label={label}
-              labelHidden
-              onChange={this.handleSelection}
-              checked={selected}
-              disabled={loading}
-            />
+            <span onChange={this.handleLargerSelectionArea}>
```

Comment:
Our Checkbox renders block elements so probably best to make this a div.

Reply:
Nice catch, I'll change this on Monday and merge 🚢